react_app/src/components/CoursePage/ReviewItemList.js

```diff
@@ -21,18 +22,25 @@ class ReviewItemList extends React.Component {
                     </div>
                 </div>
                 <div id="reviewList">
-                    {reviewItems.map((item) => {
-                        return (<ReviewItem key={item.id} date={item.date} rating={item.rating} text={item.text}
-                                            id={item.id}/>);
-                    })}
+                {Object.values(this.props.reviewList)
+                  .filter(review => review.Course_id === couresId)
```

Comment:
Is this necessary if we already expect server to return reviews for specific course?

Reply:
You're right, I've removed it.  Thanks.